### PR TITLE
Bump NuGet reference versions and add GitHub source link

### PR DESCRIPTION
**Pull request by ds5678**

### CODE REVIEW

File: NativeFileDialogs.AutoGen/NativeFileDialogs.AutoGen.csproj [hunk 1]
```diff
@@ -23,6 +23,10 @@
 
   <ItemGroup Label="Package References">
     <PackageReference Include="CppSharp.Runtime" Version="1.0.3" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```
Comment (Speykious) [inline NativeFileDialogs.AutoGen/NativeFileDialogs.AutoGen.csproj:29]: Didn't know you could do that... What does it do exactly?

File: Examples/Net/NativeFileDialogs.Net.Example.SaveDialog/NativeFileDialogs.Net.Example.SaveDialog.csproj [hunk 2]
```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup Label="Package References">
-    <PackageReference Include="NativeFileDialogs.Net.Runtime" Version="0.1.0-alpha" />
+    <PackageReference Include="NativeFileDialogs.Net.Runtime" Version="1.0.0" />
```
Comment (Speykious) [inline Examples/Net/NativeFileDialogs.Net.Example.SaveDialog/NativeFileDialogs.Net.Example.SaveDialog.csproj:15]: Bruh I didn't even think I had bumped the version in the lib packages... 💀

I guess this is alright.

File: NativeFileDialogs.AutoGen/NativeFileDialogs.AutoGen.csproj (hunk 1 above)
Comment (ds5678) [reply NativeFileDialogs.AutoGen/NativeFileDialogs.AutoGen.csproj:29]: Ctrl click in the IDE shows actual source code instead of decompiling.

Comment (Speykious) [reply NativeFileDialogs.AutoGen/NativeFileDialogs.AutoGen.csproj:29]: Woa... :0 That's amazing!

Time to include this in all my C# projects

**Review by Speykious — APPROVED** [review]
👌

**Pull request merged**